### DISCUSSION TITLE
Welcome to cornflow-server Discussions!

### DISCUSSION THREAD

**Discussion by ggsdc** [baobabsoluciones/cornflow#229] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by nmnmcc** [comment]
PR [#12748](https://github.com/outline/outline/pull/12748) adds generic presigned PUT upload support behind `AWS_S3_UPLOAD_METHOD=put` config. Tested against both Cloudflare R2 and S3-compatible storage. Default is `post` — no breaking changes for existing deployments.
